doc/API.md

```diff
@@ -935,7 +935,20 @@ standard success or error response. See
 
 #### /renter/contracts [GET]
 
-returns active contracts. Expired contracts are not included.
+returns all, active, inactive, or expired contracts.  Active contracts are
+contracts that the Renter is currently using to store, upload, and download.
+Inactive contracts are contracts where the end height of the contract is greater
+than the current block height but the contract is either !goodForUpload,
+!goodForRenew, or was renewed due to running out of funds.  Expired contracts
+are contracts with an end height less than the current block height.
```

Comment:
`!goodForUpload` and `!goodForRenew` are internal variables of siad, and not something that some person new to the api should be expected to understand. Need to spell out more explicitly that it's for contracts that are no longer being used to store data.

On that note, contracts that are `!goodForUpload && goodForRenew` should be considered active, the only inactive contracts are ones that are `!goodForRenew`.

Finally, this comment doesn't indicate what happens if none of the optional flags are passed in to fetch contracts, and so the comment needs to be expanded to explain the default behavior. I believe the default behavior should be to return active contracts, because that is what maintains compatibility with how it was before. (though maybe it should contain both active and inactive).